drivers/bluetooth/hci/h4.c

```diff
@@ -19,7 +19,7 @@
 #include <string.h>
 
 #define BT_DBG_ENABLED IS_ENABLED(CONFIG_BLUETOOTH_DEBUG_HCI_DRIVER)
-#include <bluetooth/log.h>
+#include <common/log.h>
```

Comment:
Agreed. Would it work to change this to "common/log.h". Also not that the ordering of the includes from top-down is global to local, i.e. "common/log.h" should come after all <> includes.